internal/commands/build/compose.go

```diff
@@ -29,22 +32,19 @@ func parseCompose(app *types.App, options buildOptions) (map[string]build.Option
 			continue
 		}
 
-		var tags []string
-		if service.Image != nil && *service.Image != "" {
-			tags = []string{*service.Image}
+		if service.Name == "installer" {
+			return nil, errors.New("'installer' is a reserved service name, please fix your docker-compose.yml file")
```

Comment:
Why this?

Reply:
because this is the tag used by the ~invocation~ installer image.
Anyway we changed our mind and this image now isn't tagged anymore